pkg/operator/status.go

```diff
@@ -289,9 +293,82 @@ func (optr *Operator) syncUpgradeableStatus() error {
 		coStatus.Message = "One or more machine config pools are updating, please see `oc get mcp` for further details"
 	}
 
+	// don't overwrite status if we didn't run
```

Comment:
I'm not convinced we should be reporting on a node level. In a large cluster this could  result  in a big list of nodes when 1) the MCO doesn't think via individual nodes but rather pools and 2) a long list is unmangeable as a ui

I think this should probably be reported at a pool level.

Reply:
a message like: Worker pool contains nodes with an unsupported kubelet skew. Please see `oc get nodes` for details and ensure all nodes have a kubelet Version of at least $version.

Would be more concise but also more descriptive.

Reply:
@kikisdeliveryservice  What is reporting at the pool level? The version skew should be checked for each node, follow the same implementation as https://github.com/openshift/machine-config-operator/blob/db56bd2856b4892f3b72e6056904a29676d362d4/pkg/operator/status.go#L289, the message is reported at pool level.  Could you elaborate?

Reply:
I have a suggestion above: https://github.com/openshift/machine-config-operator/pull/2658/files#r663253475